irohad/ametsuchi/tx_cache_response.hpp

```diff
@@ -23,6 +23,9 @@ namespace iroha {
        * response classes
        */
       struct HashContainer {
+        HashContainer() = default;
+        HashContainer(HashType h) : hash(h) {}
```

Comment:
Should be explicit?

Reply:
And do we really want always to accept the hash by value? Maybe a const ref ctor should also be defined?